packages/framework/fluid-framework/api-extractor.json

```diff
@@ -1,4 +1,13 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "../../../common/build/build-common/api-extractor-base.json"
+	"extends": "../../../common/build/build-common/api-extractor-base.json",
+	"bundledPackages": ["@fluidframework/map"],
+	"messages": {
+		"extractorMessageReporting": {
+			// TODO: Add missing documentation and remove this rule override
+			"ae-undocumented": {
```

Comment:
do we have bugs to get these cleaned up? maybe make a feature, since there is probably a task per package

Reply:
Not yet. I actually think I need to disable this in the root config for the time being. I've identified some bugs with this on the api-extractor side that probably need to be resolved before we can confidently light this up. I'll put up a PR for that.

Reply:
https://github.com/microsoft/FluidFramework/pull/18843